livre/eq/Node.cpp

```diff
@@ -55,9 +56,12 @@ struct Node::Impl
         const size_t maxMemBytes =
                 vrRenderParameters.getMaxCPUCacheMemoryMB() * LB_1MB;
 
-        _textureDataCache.reset( new livre::TextureDataCache( maxMemBytes,
-                                                              *_dataSource.get(),
-                                                              GL_UNSIGNED_BYTE ));
+        _textureDataCache.reset( new TextureDataCache( maxMemBytes,
+                                                       *_dataSource.get(),
+                                                       GL_UNSIGNED_BYTE ));
+        const size_t histCacheSize = 128 * LB_1MB; // Histogram cache is 128 MB
```

Comment:
It is not pre-allocating 128mb. This is the max size. That will hold lots of histograms. 512k histograms. A bit too much maybe.